models/users-models.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection')
+
+function fetchUsers() {
+    return db
+    .query('SELECT * FROM users;').then(({ rows })=> {
+        return rows
+    })
+}
+
```

Comment:
Nice, you're missing adding this endpoint to your `endpoints.json` file though 📝